test/feature-benchmark/scenarios.py

```diff
@@ -62,6 +62,34 @@ def benchmark(self) -> MeasurementSource:
         )
 
 
+class MFPPushdown(Scenario):
+    """Test MFP pushdown -- WHERE clause with a suitable condition and no index defined."""
+
+    SCALE = 7
+
+    def init(self) -> List[Action]:
+        return [
+            self.table_ten(),
+            TdAction(
+                f"""
+> CREATE MATERIALIZED VIEW v1 (f1, f2) AS SELECT {self.unique_values()} AS f1, 1 AS f2 FROM {self.join()}
+
+> SELECT COUNT(*) = {self.n()} FROM v1;
+true
+"""
+            ),
+        ]
+
+    def benchmark(self) -> MeasurementSource:
+        return Td(
+            """
+> /* A */ SELECT 1;
+1
+> /* B */ SELECT * FROM v1 WHERE f2 < 0;
```

Comment:
Should we also verify that this is actually being pushed down? Maybe once in the init?

Reply:
Yes, I was thinking about this but I want to run comparison tests where the pushdown is disabled for one of the two participting Mz instances.